src/styles/lexical-theme.module.css

```diff
@@ -60,6 +60,78 @@
   list-style:none;
 }
 
+.listitem {
```

Comment:
Can you please replace the hard-coded sizes and colors with the CSS variables from here? https://github.com/mdx-editor/editor/blob/main/src/styles/ui.module.css#L36-L132.

Reply:
Sure, let me use variable instead of the hardcoded value.